core/templates/dev/head/app.js

```diff
@@ -93,6 +93,10 @@ oppia.config(['$provide', function($provide) {
   }]);
 }]);
 
+oppia.factory('checkIfMobileDevice', [function() {
```

Comment:
Let's call it deviceInfoService or similar (we generally end service names with 'Service')